docs/api/baseplate/observers/tracing.rst

```diff
@@ -0,0 +1,36 @@
+Tracing
+=======
+
+The tracing observer reports span information to a Zipkin-compatible
```

Comment:
Worth adding a link to Zipkin?

Reply:
we don't actually use zipkin internally anymore, right? just use something that's compatible with its span ingest? I wasn't sure how much to call out the zipkin angle and confuse people.

Reply:
Correct, no more Zipkin. It sounds like the wider ecosystem might be heading in the direction of OpenTracing, but I think we are sending Zipkin-formatted traces to Wavefront (which accepts Zipkin + OpenTracing traces).